### ISSUE TITLE
confused: dml with pre-trained model and same network on Market-1501

### ISSUE THREAD

**Issue by levyfan** [YingZhangDUT/Deep-Mutual-Learning#9]
Hi, I am confused with the experiment on Market-1501 in section 3.3.

In this experiment, net1 = MobileNet, net2 = MobileNet, and they are both pre-trained model in ImageNet_Pretrain = yes (last line) settings. Thus, net1 is exactly the same with net2, right?

So, the prediction of net1 and net2 should be exactly the same because MobileNet does not have Dropout. Thus, p1 = p2, and Dkl = 0, and Loss1 = Loss2, right?

Thus, after training, net1 should be exactly the same with net2, right?

So, DML in this settings has Dkl = 0, and this has little difference with training single MobileNet. Why it has a huge performance improvement (rank-1 87.73 vs  83.94)?

The paper also says "Models Θ1 and Θ2 to different initial conditions". How does this relate to pre-trained models?

**Comment by banxia1994** [comment]
hi, have any progress about pre-trained model ?

**Comment by levyfan** [comment]
@banxia1994 I emailed the authors and they said they add Dropout which makes 2 nets different at their variables.

**Comment by banxia1994** [comment]
> @banxia1994 I emailed the authors and they said they add Dropout which makes 2 nets different at their variables.

thanks~
